bridges/bin/runtime-common/Cargo.toml

```diff
@@ -41,8 +41,8 @@ sp-std = { git = "https://github.com/paritytech/substrate", branch = "master", d
 sp-trie = { git = "https://github.com/paritytech/substrate", branch = "master", default-features = false }
 
 # Polkadot dependencies
-xcm = { git = "https://github.com/paritytech/polkadot", branch = "master", default-features = false }
-xcm-builder = { git = "https://github.com/paritytech/polkadot", branch = "master", default-features = false }
+xcm = { git = "https://github.com/paritytech/polkadot", default-features = false , branch = "rh-async-backing-feature" }
+xcm-builder = { git = "https://github.com/paritytech/polkadot", default-features = false , branch = "rh-async-backing-feature" }
```

Comment:
I think these need to be updated back to `master`? 

Maybe you're already aware and chose not to, but you can use [diener](https://github.com/paritytech/diener) to patch a local checkout of polkadot. 

You may find it easier than changing the branch in all these cargo.tomls.

```bash
# in polkadot
git checkout rh-async-backing-feature
cd ../cumulus
diener patch --crates-to-patch ../polkadot --polkadot
```

Reply:
It was updated to `rh-async-backing` using diener, and will be reverted to master before the merge.

(imo it was more convenient to point to origin, instead of local branch, so you're not locking polkadot folder)